qa/rpc-tests/fundrawtransaction.py

```diff
@@ -678,14 +678,18 @@ def run_test(self):
         assert(signedtx["complete"])
         self.nodes[0].sendrawtransaction(signedtx["hex"])
 
+        self.sync_all()
```

Comment:
I think we should get rid of such sync_all() as they pause the test suite quite a bit.